media/js/firefox/new.js

```diff
@@ -171,12 +171,41 @@
                     $thankYou.focus();
                 }, 500);
             }
+
+            // Added measurements fire only when the download is fired
+            spring_campaign_measurement();
```

Comment:
This should probably be moved into the `if (scene === 2)` block above, right?

Also, should add logic to make sure pixel is only added once. Currently, if you hit `/firefox/new`, click the CTA, then click the back button, the `spring_campaign_measurement` function is called again.